### PR TITLE
Added a name

### PR DESCRIPTION
**Pull request by vaibhav111tandon**

### CODE REVIEW

File: people.json [hunk 1]
```diff
@@ -4,5 +4,11 @@
     "github": "@RamonGebben",
     "twitter": "@RamonGebben",
     "linkedin": "https: //www.linkedin.com/in/ramon-gebben-88039b85"
+  },
+  {
+    "name": "Ramon Gebben",
```
Comment (RamonGebben) [inline people.json]: Looks like that's not your name 😂

Comment (RamonGebben) [inline people.json]: Looks like that's not your name 😂

**Review by RamonGebben — CHANGES_REQUESTED** [review]

File: people.json (hunk 1 above)
Comment (vaibhav111tandon) [reply people.json]: Oops

Comment (vaibhav111tandon) [reply people.json]: Changes done

File: people.json [hunk 2]
```diff
@@ -4,5 +4,11 @@
     "github": "@RamonGebben",
     "twitter": "@RamonGebben",
     "linkedin": "https: //www.linkedin.com/in/ramon-gebben-88039b85"
+  },
+  {
+    "name": "Vaibhav Tandon",
+    "github": "@vaibhav111tandon",
+    "twitter": "NA",
```
Comment (the-mo) [inline people.json]: `"twitter": "NA",` should be `"twitter": "N/A",`

**Review by the-mo — CHANGES_REQUESTED** [review]

File: people.json (hunk 2 above)
Comment (vaibhav111tandon) [reply people.json]: Ok

Comment (vaibhav111tandon) [reply people.json]: Done

**Review by the-mo — APPROVED** [review]

**Pull request merged**